src/Integration/Connection/ConnectionWorkflow.cs

```diff
@@ -163,13 +156,6 @@ private void OnProjectsChanged(ConnectionInformation connection, IEnumerable<Son
                         new Credential(connection.UserName, connection.Password.ToUnsecureString()));
                 }
 
-                var isCompatible = await this.AreSolutionProjectsAndSonarQubePluginsCompatibleAsync(controller, notifications,
```

Comment:
Dropped the compatibility check and all the methods and resources strings associated with it.